util/topgen/intermodule.py

```diff
@@ -387,16 +387,26 @@ def elab_intermodule(topcfg: OrderedDict):
         sig['external'] = True
 
         sig_name = port if port != "" else intersignal_format(sig)
-        sig["top_signame"] = sig_name
+
+        # if top signame already defined, then a previous connection cateogry
+        # is already connected to external pin.  Sig name is only used for
+        # port definition
+        im_connect = True if "top_signame" in sig else False
```

Comment:
it looks like this variable is only used once right below, is that right?
in that case, we can consider merging this into the if statement directly:
```Python
if 'top_signame' not in sig:
  sig["top_signame"] = sig_name
```